www/md_base/src/app/common/directives/builditem/builditem.directive.coffee

```diff
@@ -14,12 +14,10 @@ class BuildItem extends Directive
         }
 
 class _BuildItem extends Controller
-    constructor: (buildbotService) ->
+    constructor: (dataService) ->
         # TODO: add links to builders and builds after the corresponding pages has finished
         if @outBuilder
             @builder = outBuilder
         else if @showBuilder
-            buildbotService.one('builders', @build.builderid).get().then (data) =>
-                @builder = data
-
-
+            dataService.getBuilders(@build.builderid).then (builders) =>
```

Comment:
I messed this up, it was me, right? It won't clean up the listeners. You can pass in subscribe: false to it, and it won't listen for changes.